couch/datadog_checks/couch/couch.py

```diff
@@ -283,9 +296,10 @@ def _get_instance_names(self, server, instance):
     def check(self, instance):
         server = self.agent_check.get_server(instance)
 
+        config_tags = self.agent_check.get_config_tags(instance)
         max_dbs_per_check = instance.get('max_dbs_per_check', self.agent_check.MAX_DB)
         for name in self._get_instance_names(server, instance):
-            tags = ["instance:{0}".format(name)]
+            tags = config_tags + ["instance:{0}".format(name)]
```

Comment:
consider using `config_tags.append("instance:{0}".format(name))` when you add only one item to a list (will spare creating the second list)

Reply:
`config_tags` would keep old instance tags between iterations. I think we have no choice but to create a new list on every iteration.

Reply:
Yup, my bad, now I see it.